八月 25, 2019 02:10:53 +0800 [15852 1] com.newrelic INFO: Writing to New Relic log file: G:\springboot-log\newrelic\logs\newrelic_agent.log
八月 25, 2019 02:10:53 +0800 [15852 1] com.newrelic INFO: JRE vendor Oracle Corporation version 1.8.0_92
八月 25, 2019 02:10:53 +0800 [15852 1] com.newrelic INFO: JVM vendor Oracle Corporation Java HotSpot(TM) Client VM version 25.92-b14
八月 25, 2019 02:10:53 +0800 [15852 1] com.newrelic INFO: OS Windows 10 version 10.0 arch x86
八月 25, 2019 02:10:53 +0800 [15852 1] com.newrelic INFO: Agent Host: hetiantian IP: 10.0.75.1
八月 25, 2019 02:10:53 +0800 [15852 1] com.newrelic INFO: New Relic Agent v5.3.0 is initializing...
八月 25, 2019 02:10:56 +0800 [15852 12] com.newrelic INFO: Instrumentation com.newrelic.instrumentation.jdbc-resultset is disabled. Skipping.
八月 25, 2019 02:11:00 +0800 [15852 1] com.newrelic.agent.RPMServiceManagerImpl INFO: Configured to connect to New Relic at collector.newrelic.com:443
八月 25, 2019 02:11:00 +0800 [15852 1] com.newrelic INFO: Setting audit_mode to false
八月 25, 2019 02:11:00 +0800 [15852 1] com.newrelic INFO: Setting protocol to "https"
八月 25, 2019 02:11:04 +0800 [15852 1] com.newrelic.agent.config.ConfigServiceImpl INFO: Configuration file is G:\springboot-log\newrelic\.\newrelic.yml
八月 25, 2019 02:11:04 +0800 [15852 1] com.newrelic INFO: New Relic Agent v5.3.0 has started
八月 25, 2019 02:11:04 +0800 [15852 1] com.newrelic INFO: Agent class loader: com.newrelic.bootstrap.BootstrapAgent$JVMAgentClassLoader@6b179d
八月 25, 2019 02:11:04 +0800 [15852 1] com.newrelic INFO: Premain startup complete in 23,459ms
八月 25, 2019 02:11:09 +0800 [15852 1] com.newrelic INFO: Server Info: Apache Tomcat/9.0.22
八月 25, 2019 02:11:14 +0800 [15852 7] com.newrelic.agent.Agent INFO: JVM is shutting down
八月 25, 2019 02:11:14 +0800 [15852 7] com.newrelic.agent.Agent INFO: New Relic Agent has shutdown
八月 25, 2019 02:13:02 +0800 [19132 1] com.newrelic INFO: Writing to New Relic log file: G:\springboot-log\newrelic\logs\newrelic_agent.log
八月 25, 2019 02:13:02 +0800 [19132 1] com.newrelic INFO: JRE vendor Oracle Corporation version 1.8.0_92
八月 25, 2019 02:13:02 +0800 [19132 1] com.newrelic INFO: JVM vendor Oracle Corporation Java HotSpot(TM) Client VM version 25.92-b14
八月 25, 2019 02:13:02 +0800 [19132 1] com.newrelic INFO: OS Windows 10 version 10.0 arch x86
八月 25, 2019 02:13:02 +0800 [19132 1] com.newrelic INFO: Agent Host: hetiantian IP: 10.0.75.1
八月 25, 2019 02:13:02 +0800 [19132 1] com.newrelic INFO: New Relic Agent v5.3.0 is initializing...
八月 25, 2019 02:13:02 +0800 [19132 12] com.newrelic INFO: Instrumentation com.newrelic.instrumentation.jdbc-resultset is disabled. Skipping.
八月 25, 2019 02:13:06 +0800 [19132 1] com.newrelic.agent.RPMServiceManagerImpl INFO: Configured to connect to New Relic at collector.newrelic.com:443
八月 25, 2019 02:13:06 +0800 [19132 1] com.newrelic INFO: Setting audit_mode to false
八月 25, 2019 02:13:06 +0800 [19132 1] com.newrelic INFO: Setting protocol to "https"
八月 25, 2019 02:13:07 +0800 [19132 1] com.newrelic.agent.config.ConfigServiceImpl INFO: Configuration file is G:\springboot-log\newrelic\.\newrelic.yml
八月 25, 2019 02:13:07 +0800 [19132 1] com.newrelic INFO: New Relic Agent v5.3.0 has started
八月 25, 2019 02:13:07 +0800 [19132 1] com.newrelic INFO: Agent class loader: com.newrelic.bootstrap.BootstrapAgent$JVMAgentClassLoader@6b179d
八月 25, 2019 02:13:07 +0800 [19132 1] com.newrelic INFO: Premain startup complete in 8,471ms
八月 25, 2019 02:13:12 +0800 [19132 1] com.newrelic INFO: Server Info: Apache Tomcat/9.0.22
八月 25, 2019 02:13:17 +0800 [19132 30] com.newrelic INFO: Host name is hetiantian, display host hetiantian for application springboot-log
八月 25, 2019 02:13:19 +0800 [19132 30] com.newrelic INFO: Collector redirection to collector-004.newrelic.com:443
八月 25, 2019 02:13:21 +0800 [19132 30] com.newrelic INFO: Max payload size is 1,000,000 bytes
八月 25, 2019 02:13:21 +0800 [19132 30] com.newrelic INFO: Agent run id: BfTl9djZOS_VACVxSRROLyBdYX6kAAIBAAAnIQEAAEq8AgAUTi8cAwAFNS4zLjAACmhldGlhbnRpYW4ADnNwcmluZ2Jvb3QtbG9n
八月 25, 2019 02:13:21 +0800 [19132 30] com.newrelic INFO: Agent 19132@hetiantian/springboot-log connected to collector.newrelic.com:443
八月 25, 2019 02:13:21 +0800 [19132 30] com.newrelic INFO: Reporting to: https://rpm.newrelic.com/accounts/2453833/applications/340668188
八月 25, 2019 02:13:21 +0800 [19132 30] com.newrelic INFO: Using default collector host: collector.newrelic.com
八月 25, 2019 02:13:21 +0800 [19132 30] com.newrelic INFO: Using default collector host: collector.newrelic.com
八月 25, 2019 02:13:21 +0800 [19132 30] com.newrelic INFO: Real user monitoring is enabled with auto instrumentation for application "springboot-log"
八月 25, 2019 02:38:35 +0800 [19132 7] com.newrelic.agent.Agent INFO: JVM is shutting down
八月 25, 2019 02:38:36 +0800 [19132 61] com.newrelic INFO: Transaction com.newrelic.agent.Transaction@f6bf6d tried to finish but ServiceManager not started
八月 25, 2019 02:38:36 +0800 [19132 7] com.newrelic.agent.Agent INFO: New Relic Agent has shutdown
八月 25, 2019 02:40:04 +0800 [10652 1] com.newrelic INFO: Writing to New Relic log file: G:\springboot-log\newrelic\logs\newrelic_agent.log
八月 25, 2019 02:40:04 +0800 [10652 1] com.newrelic INFO: JRE vendor Oracle Corporation version 1.8.0_92
八月 25, 2019 02:40:04 +0800 [10652 1] com.newrelic INFO: JVM vendor Oracle Corporation Java HotSpot(TM) Client VM version 25.92-b14
八月 25, 2019 02:40:04 +0800 [10652 1] com.newrelic INFO: OS Windows 10 version 10.0 arch x86
八月 25, 2019 02:40:04 +0800 [10652 1] com.newrelic INFO: Agent Host: hetiantian IP: 10.0.75.1
八月 25, 2019 02:40:04 +0800 [10652 1] com.newrelic INFO: New Relic Agent v5.3.0 is initializing...
八月 25, 2019 02:40:07 +0800 [10652 12] com.newrelic INFO: Instrumentation com.newrelic.instrumentation.jdbc-resultset is disabled. Skipping.
八月 25, 2019 02:40:11 +0800 [10652 1] com.newrelic.agent.RPMServiceManagerImpl INFO: Configured to connect to New Relic at collector.newrelic.com:443
八月 25, 2019 02:40:11 +0800 [10652 1] com.newrelic INFO: Setting audit_mode to false
八月 25, 2019 02:40:11 +0800 [10652 1] com.newrelic INFO: Setting protocol to "https"
八月 25, 2019 02:40:15 +0800 [10652 1] com.newrelic.agent.config.ConfigServiceImpl INFO: Configuration file is G:\springboot-log\newrelic\.\newrelic.yml
八月 25, 2019 02:40:15 +0800 [10652 1] com.newrelic INFO: New Relic Agent v5.3.0 has started
八月 25, 2019 02:40:15 +0800 [10652 1] com.newrelic INFO: Agent class loader: com.newrelic.bootstrap.BootstrapAgent$JVMAgentClassLoader@6b179d
八月 25, 2019 02:40:15 +0800 [10652 1] com.newrelic INFO: Premain startup complete in 14,844ms
八月 25, 2019 02:40:20 +0800 [10652 1] com.newrelic INFO: Server Info: Apache Tomcat/9.0.22
八月 25, 2019 02:40:30 +0800 [10652 30] com.newrelic INFO: Host name is hetiantian, display host hetiantian for application springboot-log
八月 25, 2019 02:40:31 +0800 [10652 30] com.newrelic INFO: Collector redirection to collector-004.newrelic.com:443
八月 25, 2019 02:40:32 +0800 [10652 30] com.newrelic INFO: Max payload size is 1,000,000 bytes
八月 25, 2019 02:40:32 +0800 [10652 30] com.newrelic INFO: Agent run id: BYr3f8yx8RMaACVxSRROLyBdYYT4AAIBAAAnIQEAACmcAgQUTi8cAwAFNS4zLjAACmhldGlhbnRpYW4ADnNwcmluZ2Jvb3QtbG9n
八月 25, 2019 02:40:32 +0800 [10652 30] com.newrelic INFO: Agent 10652@hetiantian/springboot-log connected to collector.newrelic.com:443
八月 25, 2019 02:40:32 +0800 [10652 30] com.newrelic INFO: Reporting to: https://rpm.newrelic.com/accounts/2453833/applications/340668188
八月 25, 2019 02:40:32 +0800 [10652 30] com.newrelic INFO: Using default collector host: collector.newrelic.com
八月 25, 2019 02:40:32 +0800 [10652 30] com.newrelic INFO: Using default collector host: collector.newrelic.com
八月 25, 2019 02:40:33 +0800 [10652 30] com.newrelic INFO: Real user monitoring is enabled with auto instrumentation for application "springboot-log"
八月 27, 2019 10:10:59 +0800 [3600 1] com.newrelic INFO: Writing to New Relic log file: G:\java工程\springboot-log\newrelic\logs\newrelic_agent.log
八月 27, 2019 10:10:59 +0800 [3600 1] com.newrelic INFO: JRE vendor Oracle Corporation version 1.8.0_181
八月 27, 2019 10:10:59 +0800 [3600 1] com.newrelic INFO: JVM vendor Oracle Corporation Java HotSpot(TM) 64-Bit Server VM version 25.181-b13
八月 27, 2019 10:10:59 +0800 [3600 1] com.newrelic INFO: OS Windows 10 version 10.0 arch amd64
八月 27, 2019 10:10:59 +0800 [3600 1] com.newrelic INFO: Agent Host: spc-hz20121817 IP: 10.0.75.1
八月 27, 2019 10:10:59 +0800 [3600 1] com.newrelic INFO: New Relic Agent v5.3.0 is initializing...
八月 27, 2019 10:11:00 +0800 [3600 12] com.newrelic INFO: Instrumentation com.newrelic.instrumentation.jdbc-resultset is disabled. Skipping.
八月 27, 2019 10:11:07 +0800 [3600 1] com.newrelic.agent.RPMServiceManagerImpl INFO: Configured to connect to New Relic at collector.newrelic.com:443
八月 27, 2019 10:11:07 +0800 [3600 1] com.newrelic INFO: Setting audit_mode to false
八月 27, 2019 10:11:07 +0800 [3600 1] com.newrelic INFO: Setting protocol to "https"
八月 27, 2019 10:11:09 +0800 [3600 1] com.newrelic.agent.config.ConfigServiceImpl INFO: Configuration file is G:\java工程\springboot-log\newrelic\.\newrelic.yml
八月 27, 2019 10:11:09 +0800 [3600 1] com.newrelic INFO: New Relic Agent v5.3.0 has started
八月 27, 2019 10:11:09 +0800 [3600 1] com.newrelic INFO: Agent class loader: com.newrelic.bootstrap.BootstrapAgent$JVMAgentClassLoader@6433a2
八月 27, 2019 10:11:09 +0800 [3600 1] com.newrelic INFO: Premain startup complete in 11,535ms
八月 27, 2019 10:11:14 +0800 [3600 1] com.newrelic INFO: Server Info: Apache Tomcat/9.0.22
八月 27, 2019 10:11:19 +0800 [3600 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 10:11:20 +0800 [3600 30] com.newrelic INFO: Collector redirection to collector-004.newrelic.com:443
八月 27, 2019 10:11:21 +0800 [3600 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.doConnect(DataSenderImpl.java:377) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:338) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 10:11:21 +0800 [3600 30] com.newrelic INFO: Remote connect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 10:11:21 +0800 [3600 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 10:11:36 +0800 [3600 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 10:11:40 +0800 [3600 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 10:11:40 +0800 [3600 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 10:11:40 +0800 [3600 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 10:11:55 +0800 [3600 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 10:11:57 +0800 [3600 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 10:11:57 +0800 [3600 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 10:11:57 +0800 [3600 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 10:12:27 +0800 [3600 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 10:12:31 +0800 [3600 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 10:12:31 +0800 [3600 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 10:12:31 +0800 [3600 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 10:13:31 +0800 [3600 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 10:13:32 +0800 [3600 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 10:13:32 +0800 [3600 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 10:13:32 +0800 [3600 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 10:15:32 +0800 [3600 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 10:15:32 +0800 [3600 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 10:15:32 +0800 [3600 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 10:15:32 +0800 [3600 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 10:20:32 +0800 [3600 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 10:20:33 +0800 [3600 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 10:20:33 +0800 [3600 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 10:20:33 +0800 [3600 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 10:25:33 +0800 [3600 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 10:25:38 +0800 [3600 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 10:25:38 +0800 [3600 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 10:25:38 +0800 [3600 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 10:30:38 +0800 [3600 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 10:30:38 +0800 [3600 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 10:30:38 +0800 [3600 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 10:30:38 +0800 [3600 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 10:35:38 +0800 [3600 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 10:35:39 +0800 [3600 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 10:35:39 +0800 [3600 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 10:35:39 +0800 [3600 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 10:40:39 +0800 [3600 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 10:40:40 +0800 [3600 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 10:40:40 +0800 [3600 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 10:40:40 +0800 [3600 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 10:45:40 +0800 [3600 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 10:45:44 +0800 [3600 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 10:45:44 +0800 [3600 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 10:45:44 +0800 [3600 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 10:50:45 +0800 [3600 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 10:50:46 +0800 [3600 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 10:50:46 +0800 [3600 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 10:50:46 +0800 [3600 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 10:52:24 +0800 [3600 7] com.newrelic.agent.Agent INFO: JVM is shutting down
八月 27, 2019 10:52:24 +0800 [3600 7] com.newrelic.agent.Agent INFO: New Relic Agent has shutdown
八月 27, 2019 10:52:58 +0800 [18416 1] com.newrelic INFO: Writing to New Relic log file: G:\java工程\springboot-log\newrelic\logs\newrelic_agent.log
八月 27, 2019 10:52:58 +0800 [18416 1] com.newrelic INFO: JRE vendor Oracle Corporation version 1.8.0_181
八月 27, 2019 10:52:58 +0800 [18416 1] com.newrelic INFO: JVM vendor Oracle Corporation Java HotSpot(TM) 64-Bit Server VM version 25.181-b13
八月 27, 2019 10:52:58 +0800 [18416 1] com.newrelic INFO: OS Windows 10 version 10.0 arch amd64
八月 27, 2019 10:52:58 +0800 [18416 1] com.newrelic INFO: Agent Host: spc-hz20121817 IP: 10.0.75.1
八月 27, 2019 10:52:58 +0800 [18416 1] com.newrelic INFO: New Relic Agent v5.3.0 is initializing...
八月 27, 2019 10:52:59 +0800 [18416 12] com.newrelic INFO: Instrumentation com.newrelic.instrumentation.jdbc-resultset is disabled. Skipping.
八月 27, 2019 10:53:06 +0800 [18416 1] com.newrelic.agent.RPMServiceManagerImpl INFO: Configured to connect to New Relic at collector.newrelic.com:443
八月 27, 2019 10:53:06 +0800 [18416 1] com.newrelic INFO: Setting audit_mode to false
八月 27, 2019 10:53:06 +0800 [18416 1] com.newrelic INFO: Setting protocol to "https"
八月 27, 2019 10:53:07 +0800 [18416 1] com.newrelic.agent.config.ConfigServiceImpl INFO: Configuration file is G:\java工程\springboot-log\newrelic\.\newrelic.yml
八月 27, 2019 10:53:07 +0800 [18416 1] com.newrelic INFO: New Relic Agent v5.3.0 has started
八月 27, 2019 10:53:07 +0800 [18416 1] com.newrelic INFO: Agent class loader: com.newrelic.bootstrap.BootstrapAgent$JVMAgentClassLoader@6433a2
八月 27, 2019 10:53:07 +0800 [18416 1] com.newrelic INFO: Premain startup complete in 10,399ms
八月 27, 2019 10:53:11 +0800 [18416 1] com.newrelic INFO: Server Info: Apache Tomcat/9.0.22
八月 27, 2019 10:53:12 +0800 [18416 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 10:53:15 +0800 [18416 30] com.newrelic INFO: Collector redirection to collector-004.newrelic.com:443
八月 27, 2019 10:53:20 +0800 [18416 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.doConnect(DataSenderImpl.java:377) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:338) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 10:53:20 +0800 [18416 30] com.newrelic INFO: Remote connect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 10:53:20 +0800 [18416 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 10:53:30 +0800 [18416 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 10:53:35 +0800 [18416 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 10:53:35 +0800 [18416 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 10:53:35 +0800 [18416 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 10:53:50 +0800 [18416 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 10:53:51 +0800 [18416 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 10:53:51 +0800 [18416 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 10:53:51 +0800 [18416 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 10:54:21 +0800 [18416 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 10:54:21 +0800 [18416 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 10:54:21 +0800 [18416 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 10:54:21 +0800 [18416 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 10:55:22 +0800 [18416 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 10:55:23 +0800 [18416 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 10:55:23 +0800 [18416 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 10:55:23 +0800 [18416 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 10:57:23 +0800 [18416 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 10:57:26 +0800 [18416 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 10:57:26 +0800 [18416 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 10:57:26 +0800 [18416 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 11:02:26 +0800 [18416 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 11:02:27 +0800 [18416 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 11:02:27 +0800 [18416 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 11:02:27 +0800 [18416 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 11:07:27 +0800 [18416 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 11:07:28 +0800 [18416 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 11:07:28 +0800 [18416 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 11:07:28 +0800 [18416 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 11:12:28 +0800 [18416 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 11:12:36 +0800 [18416 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 11:12:36 +0800 [18416 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 11:12:36 +0800 [18416 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 11:17:31 +0800 [18416 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 11:17:32 +0800 [18416 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 11:17:32 +0800 [18416 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 11:17:32 +0800 [18416 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 11:22:32 +0800 [18416 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 11:22:33 +0800 [18416 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 11:22:33 +0800 [18416 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 11:22:33 +0800 [18416 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
八月 27, 2019 11:27:33 +0800 [18416 30] com.newrelic INFO: Host name is spc-hz20121817, display host spc-hz20121817 for application springboot-log
八月 27, 2019 11:27:34 +0800 [18416 30] com.newrelic INFO: Unable to connect to New Relic due to an SSL error. Consider enabling -Djavax.net.debug=all to debug your SSL configuration such as your trust store.
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.processLoop(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.Handshaker.process_record(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source) ~[?:1.8.0_181]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.deps.org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connectAndSend(DataSenderImpl.java:760) ~[newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.send(DataSenderImpl.java:856) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invoke(DataSenderImpl.java:698) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.invokeNoRunId(DataSenderImpl.java:693) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.parsePreconnectAndReturnHost(DataSenderImpl.java:352) [newrelic.jar:5.3.0]
	at com.newrelic.agent.transport.DataSenderImpl.connect(DataSenderImpl.java:330) [newrelic.jar:5.3.0]
	at com.newrelic.agent.RPMService.launch(RPMService.java:235) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.attemptConnection(RPMConnectionServiceImpl.java:331) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask.access$1100(RPMConnectionServiceImpl.java:98) [newrelic.jar:5.3.0]
	at com.newrelic.agent.rpm.RPMConnectionServiceImpl$RPMConnectionTask$3.run(RPMConnectionServiceImpl.java:233) [newrelic.jar:5.3.0]
	at com.newrelic.agent.util.SafeWrappers$1.run(SafeWrappers.java:27) [newrelic.jar:5.3.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_181]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source) ~[?:1.8.0_181]
	at java.security.cert.CertPathBuilder.build(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.validator.Validator.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source) ~[?:1.8.0_181]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source) ~[?:1.8.0_181]
	... 37 more
八月 27, 2019 11:27:34 +0800 [18416 30] com.newrelic INFO: Remote preconnect call failed : javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target.
八月 27, 2019 11:27:34 +0800 [18416 30] com.newrelic.agent.rpm.RPMConnectionServiceImpl INFO: Failed to connect to collector.newrelic.com:443 for springboot-log: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
